integration/helpers.go

```diff
@@ -793,12 +793,25 @@ func (i *TeleInstance) StartProxy(cfg ProxyConfig) error {
 	// Start the process and block until the expected events have arrived.
 	receivedEvents, err := startAndWait(process, expectedEvents)
 	if err != nil {
-		return trace.Wrap(err)
+		return nil, trace.Wrap(err)
+	}
+
+	// Extract and set reversetunnel.Server and reversetunnel.AgentPool upon
```

Comment:
Nit: The comment says it extracts both `Server` and `AgentPool` but the code below only actually deals with "tunnel ready" event.